web-ui/src/main/resources/catalog/js/GnEditorModule.js

```diff
@@ -28,7 +28,8 @@
   ]);
 
   // Define the translation files to load
-  module.constant('$LOCALES', ['core', 'search', 'editor']);
+  module.constant('$LOCALES', ['core', 'search', 'editor',
+    '/translation/db?type=StatusValue']);
```

Comment:
Would not be better to retrieve all translations, in case they can be used in the future to avoid having to change this? Or is very expensive in memory?

This change should be applied to the admin module (same comment as previous)?